test/mnemonic-test.js

```diff
@@ -64,4 +67,127 @@ describe('Mnemonic', function() {
     assert.strictEqual(m2.language, m1.language);
     assert.bufferEqual(m2.toSeed(), m1.toSeed());
   });
+
+  it('should return true for isMnemonic when passed a Mnemonic object', () => {
+    const m1 = new Mnemonic();
+    assert.strictEqual(Mnemonic.isMnemonic(m1), true);
+  });
+
+  it('should return false for isMnemonic when passed a string', () => {
+    const m1 = new Mnemonic();
+    assert.strictEqual(Mnemonic.isMnemonic(m1.getPhrase()), false);
+  });
+
+  it('should return the phrase from toString', () => {
+    const m1 = new Mnemonic();
+    assert.strictEqual(m1.toString(), m1.getPhrase());
+  });
+
+  it('should handle fromRaw correctly', () => {
+    const m1 = new Mnemonic();
+    const m2 = Mnemonic.fromRaw(m1.toRaw());
+    assert.strictEqual(m1.getPhrase(), m2.getPhrase());
+  });
+
+  it('should handle fromJSON correctly', () => {
+    const m1 = new Mnemonic();
+    const m2 = Mnemonic.fromJSON(m1.toJSON());
```

Comment:
To follow up from the last review comment, in tests like this I think you should actually assert that `m1.toJSON()` does what you think it does before comparing it with something. For example, what if `toJSON()` just returned null because of some bug? You can just quickly assert that the result of `toJSON()` is an object with non-null entropy and phrase maybe ?

Reply:
I added the code as you suggested. Just an insight into my original thinking here..  we are not testing toJSON(). In this case, it works, we assume it works, and we are testing, in the case that it works, what does fromJSON() do? Also, looking code coverage, toJSON() is called 11 times, so we have other coverage to be sure it works.